.github/actions/docs-preview/README.md

```diff
@@ -0,0 +1,49 @@
+## About
+
+GitHub Action to create a GitHub comment with the docs-preview
+
+* [Usage](#usage)
+  * [Configuration](#configuration)
+* [Customizing](#customizing)
+  * [inputs](#inputs)
+
+## Usage
+
+### Configuration
+
+Given the CI GitHub action:
+
+```yaml
+---
+on:
+  pull_request_target:
+    types: [opened]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  doc-preview:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: elastic/docs/.github/actions/docs-preview@current
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          repo: ${{ github.event.repository.name }}
+          guide-crumb: 'guide/en/observability/master/index.html'
+          pr: ${{ github.event.pull_request.number }}
+          
+```
+
+## Customizing
+
+### inputs
+
+Following inputs can be used as `step.with` keys
+
+| Name              | Type    | Default                     | Description                             |
+|-------------------|---------|-----------------------------|-----------------------------------------|
+| `guide-crumb`     | String  |                             | The URL crumb to append to base doc url |
```

Comment:
I don't love the usage of "crumb" here. Something like "path" is probably more accurate. 
Since all are required other than this one, maybe remove the "Default" column and update the description to match what's in `action.yml`.